src/marqo/tensor_search/validation.py

```diff
@@ -722,4 +723,22 @@ def validate_model_dimensions(input_dimensions: Any):
     raise InternalError(f"Model dimensions must be a positive integer! Given model dimensions = {input_dimensions}.")
 
 
+def validate_model_properties_no_model(model_properties: dict):
+    """
+    model_properties dict must have exactly 1 key: "dimensions".
```

Comment:
Nit: it could be nice to have a more explicit docstring name, as `no_model` in the function name may get misinterpreted.  For example:
`model_properties dict must have exactly 1 key: "dimensions". Validation when an index's model is set to no_model. `